app/controllers/moments_controller.rb

```diff
@@ -7,23 +7,20 @@ class MomentsController < ApplicationController
   include Shared
 
   before_action :set_moment, only: %i[show edit update destroy]
+  before_action :load_viewers, only: %i[new edit create update]
 
   # GET /moments
   # GET /moments.json
   # rubocop:disable MethodLength
   def index
     if current_user
-      @user_logged_in = true
-      period = 'day'
       # +1 day buffer to ensure we include today as well
       end_date = Date.current + 1.day
-      start_date = get_start_by_period(period, end_date)
-      @react_moments = Moment.where(user: current_user)
-                             .group_by_period(period,
+      start_date = end_date - 1.week
+      @react_moments = current_user.moments
+                             .group_by_period('day',
```

Comment:
Rubocop is complaining that `.group_by_period` needs to be aligned with `.moments` on line 20.